src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageMetadataControl.xaml

```diff
@@ -24,7 +24,8 @@
                     Style="{StaticResource HyperlinkStyle}"
                     Command="{x:Static nuget:PackageManagerControlCommands.OpenExternalLink}"
                     ToolTip="{Binding Link}"
-                    NavigateUri="{Binding Link}">
+                    NavigateUri="{Binding Link}"
+                    AutomationProperties.Name="{x:Static nuget:Resources.Label_License}">
```

Comment:
do you need the same thing for the license file text data template?

Reply:
I think not, because there's no label in that dialog.
Here, narrator reads "Hyperlink, Apache-2.0".
![image](https://user-images.githubusercontent.com/49205731/103805658-e3261a00-5021-11eb-968c-64384b7d296e.png)